acquisitions/src/main/resources/thunderjet/mod-orders/features/open-order-with-locations-from-different-tenants.feature

```diff
@@ -0,0 +1,109 @@
+@parallel=false
+# for https://folio-org.atlassian.net/browse/MODORDSTOR-402
+Feature: Open ongoing order
+
+  Background:
+    * url baseUrl
+    # uncomment below line for development
+#    * callonce dev {tenant: 'testorders'}
+    * callonce login testAdmin
+    * def okapitokenAdmin = okapitoken
+
+    * callonce login testUser
+    * def okapitokenUser = okapitoken
+
+    * def headersUser = { 'Content-Type': 'application/json', 'x-okapi-token': '#(okapitokenUser)', 'Accept': '*/*'  }
+    * def headersAdmin = { 'Content-Type': 'application/json', 'x-okapi-token': '#(okapitokenAdmin)', 'Accept': '*/*'  }
+
+    * configure headers = headersUser
+
+    # load global variables
+    * callonce variables
+
+    * def fundId = callonce uuid1
+    * def budgetId = callonce uuid2
+
+
+  Scenario: Create fund and budget
+    * configure headers = headersAdmin
+    * call createFund { 'id': '#(fundId)' }
+    * call createBudget { 'id': '#(budgetId)', 'allocated': 100, 'fundId': '#(fundId)', 'status': 'Active' }
```

Comment:
create Fund and budget with def v = call based on new approach

Reply:
added